tests/playwright/src/runner/podman-desktop-runner.ts

```diff
@@ -175,6 +177,7 @@ export class PodmanDesktopRunner {
     const video = this.getPage().video();
     if (video) {
       await video.saveAs(path);
+      await video.delete();
```

Comment:
This will delete the `hash.webm` file and keep named file?

Reply:
yes

Reply:
Awesome!